command/analyzer/dryrun/run.go

```diff
@@ -32,6 +32,7 @@ var (
 // The params required while running the Analysis locally.
 type AnalyzerDryRun struct {
 	Client               *docker.DockerClient   // The client to be used for all docker related ops.
+	DockerImageName      string                 // The name of the Docker image published to a registry.
```

Comment:
Change the comment to something like : `The docker image supplied by the user that needs to be pulled and run`.